### DISCUSSION TITLE
Erro ao enviar conteudo para o email

### DISCUSSION THREAD

**Discussion by taifagundes** [emailjs-com/help#40] [Q&A]
Na pagina contact, ao clicar em enviar o conteúdo que está no formulário o console mostra que foi enviado, o histórico do emailjs mostra que foi enviado com os dados, mas no email que eu cadastrei não está aparecendo o conteúdo. Eu já fiz a parte do 'my default template' corretamente, mas realmente o conteúdo não está aparecendo dentro do email enviado, eu recebo o email vazio.

**Comment by xr0master** [comment]
Make sure your dynamic variables match the variables set in the template.
